ast/src/expression/unary.rs

```diff
@@ -16,10 +16,16 @@
 
 use super::*;
 
+/// The unary operator for an unary expression.
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
 pub enum UnaryOperation {
+    /// The logical not operator, i.e., `!`.
+    /// For example, it transforms `true` to `false`.
     Not,
+    /// The negation operator, i.e., `-`.
```

Comment:
I would say arithmetic negation here, to distinguish from logical and bitwise negations.